src/utils/components.tsx

```diff
@@ -21,6 +21,10 @@ import CheckYourAnswers from '@/components/CheckYourAnswers/CheckYourAnswers';
 import { findValueFromFormData } from './formUtils';
 import Summary from '@/components/Summary/Summary';
 import CannotFindSchool from '@/components/CannotFindSchool/CannotFindSchool';
+import { notification } from 'antd';
+import LogoPurple from '@/assets/logo/LogoPurple';
+import { CloseOutlined } from '@ant-design/icons';
+import notificationStyles from '@/components/InstitutionAdminDashboard/InstitutionAdminDashboard.module.scss';
```

Comment:
Instead of importing from here, I would just create global styles in the globals file, one for notificationMessage and one for notification. Then you don't have to denote with styles.classname, you just have it the standard way with the classnames as a string
You may or may not need to import the globals file at the top of the tsx file. If you do, it'll just be
`import  '@/assets/global.scss'`